tox.ini

```diff
@@ -20,13 +21,31 @@ commands =
 
 [testenv:type]
 skipsdist = true
-skip_install = true
+setenv =
+    DJANGO_CONFIGURATION = DevelopmentConfiguration
+passenv =
```

Comment:
If we were using the Mypy plugin, then the settings file would be getting parsed and these should be necessary. However, this shouldn't be necessary just to import all the files. Do you know what is requiring these?

Reply:
Yeah that's exactly why - I was using `mypy_django_plugin` previously, before I realized it didn't work with `django-configuration` - updated

Reply:
So can this be removed now?

Reply:
My bad, I thought you were referring to the `setenv` section here, not `passenv`. But yes, `passenv` can also be removed now, I've pushed that change.